lib/jwe/decrypt.js

```diff
@@ -285,7 +285,7 @@ function JWEDecrypter(ks, globalOpts) {
       promise = promise.then(function(jwe) {
         if ("DEF" === jwe.header.zip) {
           return new Promise(function(resolve, reject) {
-            zlib.inflateRaw(Buffer.from(jwe.plaintext), function(err, data) {
+            pako.inflateRaw(Buffer.from(jwe.plaintext), function(err, data) {
```

Comment:
Tests are failing here.  From a glance, it appears `pako.inflateRaw` is synchronous, not callback-style ansynchronous.